File: python/udemy/5-estrutura-repetitiva/loops_definidos/sequencia_impares.py
#  (adaptado de URI 1067) Leia um valor inteiro X. Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X, se for o caso. 

x = int(input('Digite um número inteiro: '))

for i in range(0, x+1):
    if i % 2 != 0:
        print(i)

File: c/udemy/5-estruturas-repetitivas/loops-definidos/sequencia_impares.c
// (adaptado de URI 1067) Leia um valor inteiro X. Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X, se for o caso.

#include <stdio.h>

int main() {

    int x;

    printf("Valor de X: ");
    scanf("%d", &x);

    for (int i=1; i<=x; i++){
        if (i % 2 != 0){
            printf("%d\n", i);
        }
    }

    return 0;

}